vkconfig/mainwindow.cpp

```diff
@@ -759,18 +800,22 @@ void MainWindow::DuplicateClicked(ConfigurationListItem *item) {
 // Import a configuration file. File copy followed by a reload.
 void MainWindow::ImportClicked(ConfigurationListItem *item) {
     (void)item;  // We don't need this
-
+    PushCurrentItem();
     Configurator &configurator = Configurator::Get();
 
     QString full_suggested_path = configurator.GetPath(Configurator::LastImportPath);
     QString full_import_path =
         QFileDialog::getOpenFileName(this, "Import Layers Configuration File", full_suggested_path, "*.json");
-    if (full_import_path.isEmpty()) return;
+    {
```

Comment:
Why is this couple of lines in its own block?

Reply:
Um... too much caffeine? Looks like refactoring gone bad here. Fixed for final PR.